src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,32 @@
 import React from 'react';
+import { Post } from '../../types/post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+      {post.user && (<UserInfo user={post.user} />)}
+    </div>
+
+    <p className="PostInfo__body">
+      {post.body}
+    </p>
+    {post.comments.length > 0
```

Comment:
You can keep it simple with boolean. 0 is falsy value anyway and the length can't be less than 0
```suggestion
    {post.comments.length
```